capa/features/extractors/smda/insn.py

```diff
@@ -335,8 +335,6 @@ def extract_function_calls_from(f, bb, insn):
                 # if we found a jump target and it's the function address
                 # mark as recursive
                 yield Characteristic("recursive call"), outref
-    if insn.offset in f.apirefs:
-        yield Characteristic("calls from"), f.apirefs[insn.offset]
```

Comment:
results in `VA: kernel32.dll!lstrlenW - Feature: characteristic(calls from)`
API call extracted via API feature